python/ray/serve/schema.py

```diff
@@ -1167,6 +1204,9 @@ class ApplicationDetails(BaseModel, extra=Extra.forbid, frozen=True):
     deployments: Dict[str, DeploymentDetails] = Field(
         description="Details about the deployments in this application."
     )
+    external_scaler_enabled: bool = Field(
+        description="Whether external autoscaling is enabled for this application.",
```

Comment:
nit: not necessarily autoscaling anymore
```suggestion
        description="Whether external scaling is enabled for this application.",
```

Reply:
done